FirefoxPrivateNetworkVPNUITests/BaseTestCase.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FirefoxPrivateNetworkVPNUITests
+//  FirefoxPrivateNetworkVPNUITests
+//
+//  This Source Code Form is subject to the terms of the Mozilla Public
+//  License, v. 2.0. If a copy of the MPL was not distributed with this
+//  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+//  Copyright © 2019 Mozilla Corporation.
```

Comment:
2020?

Reply:
mm not sure... I added it as it was defined in the existing file [FirefoxPrivateNetworkVPNUITests.swift ](https://github.com/mozilla-mobile/guardian-vpn-ios/blob/develop/FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift)